### ISSUE TITLE
Context module training implementation plans

### ISSUE THREAD

**Issue by OliverColeman** [nicolov/segmentation_keras#29]
Thanks for creating this implementation. Do you have any plans to implement training of the context module (to allow producing full resolution segmentation maps)?

**Comment by cclough** [comment]
I'm really interested in training the context module too. Main reason I wish to use this model is for the high res. segmentation prediction. 

Is it complex to do - could you guide us on how to do it?

**Comment by nicolov** [comment]
I don't think it would be complex, but I didn't get around to it back when I implemented this. Do you think this is still a competitive architecture?

**Comment by cclough** [comment]
Thanks for the quick reply. Yes this fits a niche - it's a better architecture than FCN & SegNet, but doesn't require 8 GPUs to train like the latest 'RefineNet's etc. It's also such nice and simple code in Keras. I can't get, for example DeepLab V2, in Keras....

If you implement it (or guide me on how), I would love to use it.

Christian
